torch/csrc/generic/Tensor.cpp

```diff
@@ -771,12 +771,14 @@ static bool THPTensor_(_convertToTensorIndexers)(
 
     if (!PySlice_Check(item)) {
       // Returns NULL upon conversion failure
+      PyObject *obj = PySequence_Fast_GET_ITEM(fast.get(), i);
```

Comment:
This comment should be moved one line below (it's misleadingly implying that `GET_ITEM` can fail).